WordPress/Classes/Stores/StatsPeriodStore.swift

```diff
@@ -172,15 +172,38 @@ struct PeriodStoreState {
     var postStatsFetchingStatuses = [Int: StoreFetchingStatus]()
 }
 
+protocol StatsPeriodStoreDelegate: AnyObject {
+    func didChangeSpamState(for referrerDomain: String, isSpam: Bool)
+    func chagingSpamStateForReferrerDomainFailed(oldValue: Bool)
+}
+
+extension StatsPeriodStoreDelegate where Self: UIViewController {
+    func didChangeSpamState(for referrerDomain: String, isSpam: Bool) {
+        let markedText = NSLocalizedString("Marked as spam", comment: "Indicating that referrer was marked as spam").lowercased()
```

Comment:
Well not sure what's the general approach (if we need lowercase to be displayed at all, and not `www.someting.com Marked as spam`). Should localized file contain already lowercase string, or we do the modification at the call site (like i did)?

Reply:
I'd say to use the lowercase string directly instead of `lowercased` since I trust the translator to use the correct lowercased string more than I trust swifts interpretation.